geode-core/src/distributedTest/java/org/apache/geode/management/JMXMBeanReconnectDUnitTest.java

```diff
@@ -273,18 +321,57 @@ private static MBeanServerConnection connectToMBeanServer(String url) throws IOE
         .collect(toList());
   }
 
-  private void waitForLocatorsToAgreeOnMembership() {
-    await()
-        .until(
-            () -> {
-              int locator1BeanCount =
-                  getFederatedGemfireBeansFrom(locator1)
-                      .size();
-              int locator2BeanCount =
-                  getFederatedGemfireBeansFrom(locator2)
-                      .size();
-              return locator1BeanCount == locator2BeanCount;
-            });
+  /**
+   * Waits until the two locators have the same MBeans, and have NUM_BEANS beans.
+   *
+   * This method will return when the locators have the same beans and the expected number of beans.
+   * If it does not complete within GeodeAwaitility.DEFAULT_TIMEOUT seconds, a TimeoutException will
```

Comment:
Does this Javadoc need updating?  I don't see any reference to the `DEFAULT_TIMEOUT` that is claimed here, other than the actual `TIMEOUT` field incidentally also being 300 seconds.